addons/project_todo/wizard/mail_activity_todo_create.py

```diff
@@ -0,0 +1,35 @@
+from odoo import fields, models, _
+
+class MailActivityTodoCreate(models.TransientModel):
+    _name = 'mail.activity.todo.create'
+    _description = 'Create activity and todo at the same time'
+
+    summary = fields.Char('Summary')
+    date_deadline = fields.Date('Due Date', index=True, required=True, default=fields.Date.context_today)
+    user_id = fields.Many2one('res.users', 'Assigned to', default=lambda self: self.env.user, required=True, readonly=True)
+    note = fields.Html('Note', sanitize_style=True)
+
+    def create_todo_activity(self):
+        todo = self.env['project.task'].create({
+            'name': self.summary,
+            'description': self.note,
+            'date_deadline': self.date_deadline,
+            'user_ids': [self.user_id.id],
```

Comment:
```suggestion
            'user_ids': self.user_id.ids,
```
Is equivalent :slightly_smiling_face: